.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Prebuild
+
+jobs:
+  prebuild-node:
+    name: Upload Node Prebuild
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm install
+      - name: build release
+        run: |
+          npx prebuild -r node -t 10.20.0 -t 12.0.0 -t 14.0.0 --include-regex 'better_sqlite3.node$' -u ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does `npm run build-release` not need to be run here, as it's used before the Electron prebuilds?

Reply:
I was not sure if that's necessary. I am not familiar with the `lzz` that you are using in your build system. I've removed the `npm run build-release ` from both.